index.html

```diff
@@ -14,6 +14,59 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header">
+        
+        <img src="./img\karma-logo.svg" alt="karma-logo" id="logo"/>
+        <nav class="navbar">
+            <ul class="list-nav">
+                <li><b><a href="#">Meet Karma</a> </b></li>
+                <li><a href="#">How it work</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+        <img src="img\menu-hamburger.svg" alt="menu-hamburger" id="menu">
+     
+    
+</header>
+<main>
+    <section class="section-1">
+        <h2> Introducing Karma</h2>
+        <p> Bring WiFi with you, everywhere you go.</p>
+        <button> Learn More</button>
+    </section>
+    <section class="section-2">
+         <h2 class="h-font">Everyone Needs a little Karma</h2>
+         <div class="section3">
+             <div>
+                 <img src="\img\icon-devices.svg" alt="icon-devices"/>
+                 <p>Internet or all adevices</p>
+             </div>
+             <div>
+                <img src="\img\icon-coffee.svg" alt=""/>
+                <p>Boost your productivity </p>
+            </div>
+            <div>
+                <img src="\img\icon-refill.svg" alt=""/>
+                <p>Pay as You Go</p>
+            </div>
+         </div>
+         
+    </section>
+</main>
+<footer class="footer">
+    
+    <p>Join us on</p>
+    <div class="footer-img">
+        <img  src="\img\twitter-icon.svg" alt="twitter"/>
+        <img src="\img\facebook-icon.svg" alt="facebook-icon"/>
+        <img src="\img\instagram-icon.svg" alt="instagram-icon" id="inst"/>
```

Comment:
```suggestion
        <img src="\img\instagram-icon.svg" alt="Instagram"/>
```
The ID isn't used so we should remove it